tools/distrib/fix_build_deps.py

```diff
@@ -0,0 +1,410 @@
+#!/usr/bin/env python3
+
+# Copyright 2022 gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from doctest import SKIP
+import os
+import re
+import sys
+import collections
+import subprocess
+
+# find our home
+ROOT = os.path.abspath(os.path.join(os.path.dirname(sys.argv[0]), '../..'))
+os.chdir(ROOT)
+
+Vendor = collections.namedtuple('Vendor', ['name', 'score'])
```

Comment:
This is definitely the first time I've seen someone on the gRPC team outside of the Python subteam use `collections.namedtuple`. :+1: